docs/server/source/schema/vote.rst

```diff
@@ -0,0 +1,131 @@
+..  This file was auto generated by generate_schema_documentation.py
+
+===========
+Vote Schema
+===========
+
+Vote
+----
+
+A Vote is an endorsement of a Block (identified by a hash) by
+a node (identified by a public key).
+
+The outer Vote object contains the details of the vote being made
+as well as the signature and identifying information of the node
+passing the vote.
+
+
+Vote.node_pubkey
+^^^^^^^^^^^^^^^^
+
+**type:** string
+
+Ed25519 public key identifying the voting node.
+
+
+
+Vote.signature
+^^^^^^^^^^^^^^
+
+**type:** string
+
+Ed25519 signature of the `Vote Details`_ object.
+
+
+Vote.vote
+^^^^^^^^^
+
+**type:** object
+
+`Vote Details`_ to be signed.
+
+
+
+
+
+Vote Details
+------------
+
+`Vote Details`_ to be signed.
+
+
+Vote.previous_block
+^^^^^^^^^^^^^^^^^^^
+
+**type:** string
+
+SHA3 identifier of the block that preceeds the block being voted on.
```

Comment:
If you want to keep "SHA3" in the description, then that's okay, but you should make it clear that the "SHA3 identifier of the block" is the same thing as the `id` of the block. Also, I think it's important to note that this is the opinion of the voting node, not some absolute truth. How about:

`id` (SHA3 hash) of the block that precedes the block being voted on (as determined by the voting node).

(I also fixed the spelling of "precedes".)

I have similar comments for the `Vote.voting_for_block` below, i.e.

`id` (SHA3 hash) of the block being voted on.